lib/elixir/lib/exception.ex

```diff
@@ -1079,6 +1079,27 @@ defmodule BadArityError do
 end
 
 defmodule UndefinedFunctionError do
+  @moduledoc """
+  An exception raised when a function is invoked that is not defined.
+
+  The following fields of this exception are public and can be used freely:
```

Comment:
(I'm not sure if it's better, feel free to discard)
```suggestion
  The following fields of this exception are public and can be accessed freely:
```